plugin/advisor/config/config.go

```diff
@@ -0,0 +1,90 @@
+package config
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+
+	"github.com/bytebase/bytebase/plugin/advisor"
+	"gopkg.in/yaml.v3"
+)
+
+//go:embed sql-review.yaml
+var sqlReviewConfigStr string
+
+// MergeSQLReviewRules will merge the input YML config into default template.
+func MergeSQLReviewRules(ymlStr string) ([]*advisor.SQLReviewRule, error) {
+	config := SQLReviewConfiguration{}
+
+	if err := yaml.Unmarshal([]byte(sqlReviewConfigStr), &config); err != nil {
+		return nil, err
+	}
+
+	sqlReviewConfigUpdate := SQLReviewConfigUpdate{}
```

Comment:
Is it possible to reuse the same template format? Since we are essentially merging two templates together. Only the direction matters the result, but the templates seem can share the same structure?